html/semantics/embedded-content/the-object-element/usemap-casing.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>object usemap case-sensitive</title>
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/infrastructure.html#rules-for-parsing-a-hash-name-reference">
+<!-- See also: https://github.com/whatwg/html/issues/1666 -->
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div id="log"></div>
+
+<object data="/images/threecolors.png" usemap="#sImPlE" width="300" height="300"></object>
```

Comment:
Can you add a test with case-sensitive match, as a sanity test (with the opposite pass condition)? And similarly for img and radio button tests.

(It appears that Safari passes all the `object` tests but if I add another with a case-sensitive match, it still returns the `object` element rather than the `area`.)